src/translator_ingest/ingests/pathbank/pathbank.py

```diff
@@ -0,0 +1,1648 @@
+            knowledge_level=KnowledgeLevelEnum.knowledge_assertion,
+            agent_type=AgentTypeEnum.manual_agent,
+        )
+        edges.append(same_as_edge)
+
+    # Create has_participant edge from pathway to compound
+    pathway_curie = f"PathBank:{pathway_id}"
+    has_component_edge = Association(
+        id=entity_id(),
+        subject=pathway_curie,
+        predicate="biolink:has_participant",
+        object=pwc_curie,
+        sources=build_association_knowledge_sources(
+            primary=INFORES_PATHBANK,
+        ),
+        knowledge_level=KnowledgeLevelEnum.knowledge_assertion,
+        agent_type=AgentTypeEnum.manual_agent,
+    )
+    edges.append(has_component_edge)
+
+    return nodes, edges, {pwc_id: compound_translator}
+
+
+def _create_protein_node_and_edges(
+    protein: dict[str, Any], pathway_id: str
+) -> tuple[list[NamedThing], list[Association], dict[str, dict[str, str]]]:
+    """Create Protein node and same_as edges for a protein.
+
+    Returns:
+        Tuple of (nodes, edges, translator_dict) where translator_dict maps protein_id to {equiv_id: prefix}
+    """
+    nodes = []
+    edges = []
+
+    # Extract protein data
+    protein_id_raw = protein.get("id", {})
+    if isinstance(protein_id_raw, dict):
+        pwp_id = protein_id_raw.get("#text", "")
+    else:
+        pwp_id = str(protein_id_raw) if protein_id_raw else ""
+
+    if not pwp_id:
+        return [], [], {}
+
+    name = _normalize_xml_value(protein.get("name"))
+    uniprot_id_raw = _normalize_xml_value(protein.get("uniprot-id"))
+    drugbank_id_raw = _normalize_xml_value(protein.get("drugbank-id"))
+
+    # Normalize external IDs (strip any existing prefixes)
+    uniprot_id = _normalize_external_id(uniprot_id_raw, "UniProtKB")
+    drugbank_id = _normalize_external_id(drugbank_id_raw, "DRUGBANK")
+
+    # Create PathBank protein node
+    pwp_curie = f"PathBank:Protein:{pwp_id}"
+    protein_node = Protein(
+        id=pwp_curie,
+        name=name,
+    )
+    nodes.append(protein_node)
+
+    # Build translator dictionary: {equiv_id: prefix}
+    protein_translator = {pwp_curie: "PathBank"}
+    if uniprot_id:
+        protein_translator[uniprot_id] = "UniProtKB"
+    if drugbank_id:
+        protein_translator[drugbank_id] = "DRUGBANK"
+
+    # Create same_as edges FROM PathBank node TO external IDs (unidirectional)
+    # Also create minimal nodes for external IDs to satisfy normalization requirements
+    if uniprot_id:
+        uniprot_curie = f"UniProtKB:{uniprot_id}"
+        # Create minimal node for external UniProtKB ID (required for edge normalization)
+        uniprot_node = Protein(
+            id=uniprot_curie,
+            name=name,  # Use same name as PathBank protein
+        )
+        nodes.append(uniprot_node)
+        # Create edge FROM PathBank TO UniProtKB
+        same_as_edge = Association(
+            id=entity_id(),
+            subject=pwp_curie,
+            predicate="biolink:same_as",
```

Comment:
@mbrush @EvanDietzMorris - another way to describe xrefs!  -- it might be time to have this conversation in a Standards call.